pkg/apiserver/rest/dualwriter.go

```diff
@@ -203,9 +210,17 @@ func SetDualWritingMode(
 		}
 	}
 
+	if desiredMode == currentMode {
```

Comment:
We could make the condition for the syncing the following:

```
if (desiredMode == Mode3) && (currentMode == Mode2)
```

Reply:
Oh! I see. I will change it